src/tests/test_run.py

```diff
@@ -74,6 +74,10 @@ def DataFrame(cls):
         def PolarsDataFrame(cls):
             return polars.DataFrame()
 
+        @classmethod
+        def fetch_results(self, fetch_all=False):
+            ...
+
```

Comment:
is this some test code that should be deleted?

Reply:
It's for the `mock_resultset`. I had to add `fetch_results` to avoid `AttributeError`. 

Replaced `...` with `pass`. Please let me know if it's better.